parm/model_configure

```diff
@@ -1,6 +1,5 @@
 total_member:            1
 PE_MEMBER01:             {{ PE_MEMBER01 }}
-print_esmf:              {{ print_esmf }}
```

Comment:
This flag is not necessary in the 'model_configure' file any more. In the old version of the ufs weather model, this flag was necessary in 'model_configure'. Now it can be controlled in 'nems.configure'.